iothub/device/tests/IotHubDeviceClientTests.cs

```diff
@@ -1196,63 +1165,14 @@ public async Task IotHubDeviceClient_SendTelemetryAsync_WithoutExplicitOpenAsync
             await act.Should().ThrowAsync<InvalidOperationException>();
         }
 
-        [TestMethod]
-        public async Task IotHubDeviceClient_SendTelemetryAsync_BeforeExplicitOpenAsync_ThrowsInvalidOperationException()
```

Comment:
This test is not different than the previous one, because OpenAsync will never get called due to the throw from SendTelemetryAsync.